swarmrl/observables/particle_sensing.py

```diff
@@ -82,7 +84,7 @@ def initialize(self, colloids: List[Colloid]):
             [colloid.pos for colloid in colloids if colloid.type == self.sensing_type]
         )
 
-        out_indices, field_values = self.observable_fn(
+        out_indices, _, field_values = self.observable_fn(
             np.array(indices), np.array(positions), test_points, historic_values
```

Comment:
The variable name "test_points" was not clear to me. One could think about renaming that to a more intuitive name. Something like "sensing_points", "sensor_points" or "sensed_points"

Reply:
Resolved in https://github.com/SwarmRL/SwarmRL/pull/20/commits/63ab2e0d3b0653d9db8cfd473644b14e7a887a3c